.gitignore

```diff
@@ -1,3 +1,2 @@
 /vendor
-composer.lock
-composer.phar
\ No newline at end of file
+composer.phar
```

Comment:
I was going to write a long comment but [this StackOverflow answer](http://stackoverflow.com/a/18935087/3533651) sums it well.
Seldaek says that commiting it or not doesn't really matter but the next answer points out that it's useful for debugging to commit it and it doesn't provide a single downside.
Don't worry it will be my last comment about that :stuck_out_tongue_closed_eyes: I just tried to convince you, just tell me your last decision :wink: